### ISSUE TITLE
CatBoost can't handle both silent and verbose switches.

### ISSUE THREAD

**Issue by alaneckhardt** [HunterMcGushion/hyperparameter_hunter#80]
Using catboost==0.9.1.1. When used in 
```python
    experiment = CrossValidationExperiment(
        model_initializer=CatBoostRegressor,
        model_init_params=dict(iterations=10, learning_rate=0.01, depth=7, allow_writing_files=False),
        model_extra_params=dict()  # Send kwargs to `fit` and other extra methods
    )
```
it breaks down with
```
_catboost.CatboostError: Only one of the parameters silent and verbose should be set
```

**Comment by alaneckhardt** [comment]
I changed this:
https://github.com/HunterMcGushion/hyperparameter_hunter/blob/master/hyperparameter_hunter/models.py#L156

to look like this:
```python
        if "silent" in expected_fit_parameters and 'verbose' not in fit_kwargs:
            fit_kwargs["silent"] = True
```
but I'm not sure if it won't break down other models behaviour.

**Comment by alaneckhardt** [comment]
No, that's perfectly OK. Thanks for quick merges!

**Comment by HunterMcGushion** [comment]
This is closed by 824024ffa049b144d8118c5ba2c544870be9e026 (PR #84). @alaneckhardt, if you disagree, feel free to let me know, and we can reopen this! Thank you for your contributions!

**Comment by alaneckhardt** [comment]
I'm glad everything turned out well!
